Sidney-/slugtrade
# Linter

New functionality:

- Added pylint to the virtual environment.
- Created a bash script that runs pylint on a file with our custom lint settings.

How to test:

- Run venv_setup.sh to update the virtual environment.
- Run pylint.sh on the file slug_trade_app/apps.py and verify code rating of 10.00/10.00.
- Run pylint.sh on the file slug_trade_app/views.py and verify code rating of 2.77/10.00.
